src/app/workspace/[workspaceId]/issues/page.tsx

```
@@ -19,6 +19,11 @@ const IssuesPage = () => {
 	useDocumentTitle("Issue");
 
 	const workspaceId = useWorkspaceId();
+
+	if (!workspaceId) {
+		return null;
+	}
+
 	const { data: currentMember, isLoading: isMemberLoading } = useCurrentMember({
```

Biome (2.4.15)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)